example/example.json

```diff
@@ -20,6 +20,24 @@
     }
   ],
   "paths": {
+    "/api/v2/birds": {
+      "get": {
+        "responses": {
+          "200": {
+            "description": "Success",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "$ref": "#/components/schemas/Bird"
+                }
+              }
+            }
+          }
+        },
+        "summary": "Get all birds",
+        "operationId": "getBirds"
+      }
+    },
```

Comment:
This and the two new schema for `Animal` and `Bird` should be the only meaningful changes to this spec, but when I regenerated it seemed to make some unrelated changes - apologies for that.